react/features/base/testing/components/TestConnectionInfo.js

```diff
@@ -184,6 +192,9 @@ class TestConnectionInfo extends Component<Props, State> {
                 <TestHint
                     id = 'org.jitsi.meet.conference.joinedState'
                     value = { this.props._conferenceJoinedState } />
+                <TestHint
+                    id = 'org.jitsi.meet.conference.grantModeratorEnabled'
```

Comment:
Can you please call this `org.jitsi.meet.conference.grantModeratorAvailable` ? There is no way to disable it so I think it makes a bit more semantic sense.

Reply:
Updated! :D